### ISSUE TITLE
Social media links not labeled

### ISSUE THREAD

**Issue by cynthiaguilas** [Virtualahan/virtualahan-com#155]
The social media section is not labeled. I think there is an icon displayed on that section. However, JAWS wasn’t able to pick it up. Same goes for the email link next to it.
